pkg/subctl/operator/common/embeddedyamls/generators/yamls2go.go

```diff
@@ -98,7 +98,7 @@ func main() {
 	//    type: string`
 
 	re := regexp.MustCompile("`([^`]*)`")
-	reNS := regexp.MustCompile(`(?s)\s*namespace:\s*placeholder\s*`)
+	reNS := regexp.MustCompile(`(?s)\s*\$(SA_NAMESPACE):\s*placeholder\s*`)
```

Comment:
I think this is not the same.

The previous regex was looking for 
namespace: placeholder


Now you are looking for 

$(SA_NAMESPACE): placeholder

This needs testing, but what about:

```suggestion
	reNS := regexp.MustCompile(`(?s)\s*namespace:\s*\$(SA_NAMESPACE)\s*`)
```

Reply:
Thanks for paying attention :+1: